.github/workflows/track-no-cover.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Check no cover Removal
+
+on:
+  workflow_dispatch:
+  schedule:
```

Comment:
In practice, how are we going to notice that this has failed?

Reply:
We'll get a failed build, which generates notifications for anyone watching the repo.